### DISCUSSION TITLE
A new version tag

### DISCUSSION THREAD

**Discussion by abcdw** [magit/transient#225] [Ideas]
Hi @tarsius! Could you place a new version tag, please? It will make it easier to update package definition to new version in package managers like GNU Guix.

**Comment by tarsius** [comment]
I know, I know. I am going to do it in January. Sorry I cannot do it earlier.

**Reply by abcdw** [reply]
Thank you very much! No pressure, I just walking around, post such issues in various trackers and update related packages in guix and rde repos :) Currently docker.el, which relies on unreleased transient is broken, I workarounded it in rde, but not in guix.

BTW, we updated magit in guix repo to unreleased version because there are some neat fixes and improvements that hard to live without, but we still try to stick with upstream releases in most cases.
